runtime/doc/news.txt

```diff
@@ -317,8 +317,8 @@ The following new APIs and features were added.
 
 • |nvim_input_mouse()| supports mouse buttons "x1" and "x2".
 
-• |v_Q-default| and |v_@-default| repeat a register for each line of a visual
-  selection.
+• |v_Q-default| and |v_@-default| repeat a register for each line of a linewise
+  visual selection.
```

Comment:
👍  constraining to linewise is a useful, intuitive approach for various mappings. For example I have mapped linewise visual `D` to diff visual selections: https://github.com/justinmk/config/blob/94629007e99ad99b40166b58c7a32ba98576cb47/.config/nvim/init.vim#L257-L258

We should leverage this for more cases in the future.